processing/src/main/java/org/apache/carbondata/processing/newflow/CarbonDataLoadConfiguration.java

```diff
@@ -0,0 +1,185 @@
+package org.apache.carbondata.processing.newflow;
+
+import java.util.Iterator;
+
+import org.apache.carbondata.core.carbon.AbsoluteTableIdentifier;
+
+public class CarbonDataLoadConfiguration {
```

Comment:
Ok. I will keep the configuration but only keep important and global configurations I will keep in it, remaining configurations we can move to `Map` and keep it inside configuartion it self.  what do you say?